packages/manager/apps/web/client/app/domain/zone/all-dom-zone-activate/activate.routing.js

```diff
@@ -1,17 +1,4 @@
 export default /* @ngInject */ ($stateProvider) => {
-  $stateProvider.state('app.domain.product.zone.activate', {
-    url: '/activate',
-    views: {
-      modal: {
-        component: 'domainZoneActivate',
-      },
-    },
-    layout: 'modal',
-    resolve: {
-      goBack: /* @ngInject */ (goToZone) => goToZone,
-    },
-  });
-
   $stateProvider.state('app.domain.alldom.zone.activate', {
```

Comment:
I'm having trouble understanding why this applies on domains outside of alldom but not on domains belonging to a alldom. Is that a product specification ?

Reply:
thx for the commend, I have added it to allDom as well